integration_tests/integration/addRestrictedPatient.cy.ts

```diff
@@ -88,61 +89,65 @@ context('Add prisoner', () => {
     cy.signIn()
   })
 
-  it('should display the search for prisoner page from the homepage', () => {
-    const page = homepage.goTo()
+  it('should display the search for prisoner page from the HomePage', () => {
+    const page = Page.verifyOnPage(HomePage)
     page.addRestrictedPatient().click()
-    PrisonerSearchPage.verifyOnPage()
+    Page.verifyOnPage(PrisonerSearchPage)
   })
 
   it('Completes a add prisoner journey', () => {
     cy.visit('/add-restricted-patient/search-for-prisoner')
-    const prisonerSearchPage = PrisonerSearchPage.verifyOnPage()
-    const prisonerSearchPageForm = prisonerSearchPage.form()
```

Comment:
I've removed all the `form()` stuff - all functions now top level instead which means less variables kicking around